src/example/heat_equation_sphere/heat_equation_sphere.jl

```diff
@@ -0,0 +1,215 @@
+    x, y, z = xyz
+
+    r = norm(xyz)
+
+    θ = acos(z / r)
+    ϕ = sign(y) * acos(x / sqrt(x^2 + y^2))
+
+    return SA[r, θ, ϕ]
+end
+
+function rotMat(θx, θy, θz)
+    Rx = @SMatrix[
+        1.0 0.0 0.0
+        0.0 cos(θx) sin(θx)
+        0.0 (-sin(θx)) cos(θx)
+    ]
+    Ry = @SMatrix[
+        cos(θy) 0.0 (-sin(θy))
+        0.0 1.0 0.0
+        sin(θy) 0.0 cos(θy)
+    ]
+    Rz = @SMatrix[
+        cos(θz) sin(θz) 0.0
+        -sin(θz) cos(θz) 0.0
+        0.0 0.0 1.0
+    ]
+    return Rx * Ry * Rz
+end
+
+mutable struct VtkHandler
+    basename::Any
+    ite::Any
+    mesh::Any
+    θ::Any
+    θ_centers::Any
+    θ_vertices::Any
+    ϕ::Any
+    ϕ_centers::Any
+    ϕ_vertices::Any
+    ν::Any
+    ν_centers::Any
+    ν_vertices::Any
+    function VtkHandler(basename, mesh)
+        θ = PhysicalFunction(x -> cart2sphere(x)[2])
+        θ_centers = var_on_centers(θ, mesh)
+        θ_vertices = var_on_vertices(θ, mesh)
+
+        ϕ = PhysicalFunction(x -> cart2sphere(x)[3])
+        ϕ_centers = var_on_centers(ϕ, mesh)
+        ϕ_vertices = var_on_vertices(ϕ, mesh)
+
+        ν = Bcube.CellNormal(mesh)
+        ν_centers = transpose(var_on_centers(ν, mesh))
+        ν_vertices = transpose(var_on_vertices(ν, mesh))
+
+        new(
+            basename,
+            0,
+            mesh,
+            θ,
+            θ_centers,
+            θ_vertices,
+            ϕ,
+            ϕ_centers,
+            ϕ_vertices,
+            ν,
+            ν_centers,
+            ν_vertices,
+        )
+    end
+end
+
+function append_vtk(vtk, u::Bcube.AbstractFEFunction, t)
+    values_vertices = var_on_vertices(u, vtk.mesh)
+    values_centers = var_on_centers(u, vtk.mesh)
+
+    ## Write
+    Bcube.write_vtk(
+        vtk.basename,
+        vtk.ite,
+        t,
+        vtk.mesh,
+        Dict(
+            "u_centers" => (values_centers, VTKCellData()),
+            "u_vertices" => (values_vertices, VTKPointData()),
+            "θ_centers" => (vtk.θ_centers, VTKCellData()),
+            "θ_vertices" => (vtk.θ_vertices, VTKPointData()),
+            "ϕ_centers" => (vtk.ϕ_centers, VTKCellData()),
+            "ϕ_vertices" => (vtk.ϕ_vertices, VTKPointData()),
+            "ν_centers" => (vtk.ν_centers, VTKCellData()),
+            "ν_vertices" => (vtk.ν_vertices, VTKPointData()),
+        ),
+        ;
+        append = vtk.ite > 0,
+    )
+
+    ## Update counter
+    vtk.ite += 1
+end
+
+function run(;
+    α = 1.0 / 42,
+    degree = 1,
+    tfinal = 1,
+    nout = 100,
+    lc = 1e-1,
+    CFL = 0.5,
+    vtk_output = true,
+)
+
+    # Settings
+    out_dir = joinpath(@__DIR__, "../../../myout/heat_eqn_sphere")
+    mkpath(out_dir)
+
+    # Mesh
+    mesh_path = joinpath(out_dir, "mesh.msh")
+    Bcube.gen_sphere_mesh(mesh_path; radius = 1.0, lc = lc)
+    mesh = read_msh(mesh_path)
+    rng = MersenneTwister(0)
+    R = rotMat(rand(rng, 3)...)
+    transform!(mesh, x -> R * x) # rotate to avoid being "aligned" with an axis
```

Comment:
Why is this required?

Reply:
Good question. The coordinate mapping is "discontinuous", the angles jumping from 0 to 2pi. If the mesh is aligned with an axis, the dofs lying on the axis are not correctly computed ~ you see the discontinuity.